spec/lib/metasploit/framework/login_scanner/http_spec.rb

```diff
@@ -31,13 +31,34 @@
   end
 
   describe '#set_http_trace_proc' do
-    it 'configures http tracing' do
+    let(:sample_request) {
+      "GET / HTTP/1.1\nHost: www.google.com"
+    }
+    let(:sample_response) {
+      "HTTP/1.1 302 Found\nLocation: https://www.google.com/?gws_rd=ssl"
+    }
+    let(:expected_output) {
+      "####################\n# Request : \n####################\nGET / HTTP/1.1\nHost: www.google.com\n####################\n# Response : \n####################\nHTTP/1.1 302 Found\nLocation: https://www.google.com/?gws_rd=ssl\n"
+    }
+    let(:nil_response_output) {
+      "####################\n# Request : \n####################\nGET / HTTP/1.1\nHost: www.google.com\n####################\n# Response : \n####################\nNo response received\n"
+    }
+
+    it 'returns a proc object when HttpTrace is set to true' do
       expect(subject.set_http_trace_proc(true)).to be_kind_of(Proc)
     end
```

Comment:
This should be moved down to your `describe '#set_http_trace_proc' do` part as its related to the tests located there.